Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -221,6 +221,25 @@ - (void)updateFoV:(ARCamera*)camera {
     float fov =  atanf(1 / yScale);
     frameView.FieldOfView.AngleDown = -(frameView.FieldOfView.AngleUp = fov);
     frameView.FieldOfView.AngleLeft = -(frameView.FieldOfView.AngleRight = fov * aspectRatio);
+    
+    // Angles for FieldOfView respect the viewport ratio
+    // but tangent is not a linear function and ratio of values(l,r,b,t) computed
```

Comment:
Thanks for the fix!  I'm guessing we also need to apply this compensation on the Android side as well.  I'll add checking this to #230.